radio/src/opentx.cpp

```diff
@@ -414,8 +422,9 @@ void modelDefault(uint8_t id)
 #endif
 
 #if defined(PXX) && defined(CPUARM)
-  modelHeaders[id].modelId = g_model.header.modelId = id+1;
-  checkModelIdUnique(id);
+  modelHeaders[id].modelId[1] = g_model.header.modelId[1] = 0;
+  modelHeaders[id].modelId[0] = g_model.header.modelId[0] = id+1;

```

Comment:
Is the size of  modelHeaders[id].modelId array always 2 here? Or should you use NUM_MODULES?